examples/atlas-organization/organization-step-1/Readme.md

```diff
@@ -0,0 +1,76 @@
+# Example - A basic example to create and Orgnization with MongoDB Atlas and Terraform
```

Comment:
I think we're missing a README.MD in the root of `atlas-organization` that links together the two steps & explains the overall flow & why it's a 2-step process

Reply:
Also a question for the 2 steps - could we instead have made 2 modules in the example and then called them from a root module `main.tf` passing parameters between them as necessary? IT would save the user needing to do two `terraform apply`, but I may be missing something in the staging of the process here.

Reply:
Also a question for the 2 steps - could we instead have made 2 modules in the example and then called them from a root module `main.tf` passing parameters between them as necessary? IT would save the user needing to do two `terraform apply`, but I may be missing something in the staging of the process here.

Reply:
Yep was trying to keep them isolated so the do not try to use old API keys on new org in next apply needs work I agree

Reply:
Resolved for me, @colm-quinn leaving open as it's your point.